packages/babel-helper-compilation-targets/package.json

```diff
@@ -26,10 +26,8 @@
     "browserslist": "^4.14.5",
     "semver": "condition:BABEL_8_BREAKING ? ^7.3.4 : ^6.3.0"
   },
-  "peerDependencies": {
-    "@babel/core": "^7.0.0"
-  },
```

Comment:
was removed the change to avoid circular dependency in references with `@babel/core` which depends on this package

---

reverted it, also updated `scripts/generators/tsconfig.js` to not generate references for `peerDependencies` (tested that on - looks to be ok to skip it)

cc @nicolo-ribaudo